src/victory-label/victory-label.js

```diff
@@ -212,7 +212,12 @@ export default class VictoryLabel extends React.Component {
   renderElements(props, content) {
     const transform = this.getTransform(props);
     return (
-      <text {...props} transform={transform}>
+      <text dx={props.dx} dy={props.dy} x={props.x} y={props.y}
```

Comment:
This works fine, but take a look at this lodash method https://lodash.com/docs#pick